lib/include/jxl/encode.h

```diff
@@ -251,6 +251,24 @@ typedef enum {
    */
   JXL_ENC_OPTION_MODULAR_PREDICTOR = 27,
 
+  /** Fraction of pixels used to learn MA trees as a percentage. -1 = default,
+   * 0 = no MA and fast decode, 50 = the default, 100 = All.
```

Comment:
Can also mention that higher values will use more encode memory.

Reply:
Done